redash/query_runner/cass.py

```diff
@@ -9,13 +9,14 @@
 try:
     from cassandra.cluster import Cluster
     from cassandra.auth import PlainTextAuthProvider
+    from cassandra import ReadTimeout
     enabled = True
 except ImportError:
     enabled = False
 
 
 class Cassandra(BaseQueryRunner):
-    noop_query = "SELECT * FROM system"
+    noop_query = "SELECT * FROM system.compactions_in_progress"
```

Comment:
No  `SELECT 1` for CQL.
Most of test queries goes in the same manner like `SELECT now() FROM system.local;`